docs/src/main/asciidoc/security-basic-authentication-tutorial.adoc

```diff
@@ -46,46 +46,61 @@ You can find the solution in the `security-jpa-quickstart` link:{quickstarts-tre
+Both xref:hibernate-reactive.adoc[Hibernate Reactive] and xref:hibernate-reactive-panache.adoc[Hibernate Reactive with Panache] can be used with the `security-jpa-reactive` extension.
+
+You must also add your preferred database connector library.
+The instructions in this example tutorial use a PostgreSQL database for the identity store.
+====
+
 You can either create a new Maven project with the Security Jakarta Persistence extension or you can add the extension to an existing Maven project.
 
-* To create the Maven project with Hibernate ORM, use the following command:
+=== Create the Maven project
+
+.Create a new Maven project with the Jakarta Persistence extension
 
+* To create the Maven project with Hibernate ORM, use the following command:
++
+====
 :create-app-artifact-id: security-jpa-quickstart
 :create-app-extensions: security-jpa,jdbc-postgresql,resteasy-reactive,hibernate-orm-panache
 include::{includes}/devtools/create-app.adoc[]
-
+====
 * To create the Maven project with Hibernate Reactive, use the following command:
-
++
+====
 :create-app-artifact-id: security-jpa-reactive-quickstart
 :create-app-extensions: security-jpa-reactive,reactive-pg-client,resteasy-reactive,hibernate-reactive-panache
 include::{includes}/devtools/create-app.adoc[]
-
-[NOTE]
 ====
-xref:hibernate-orm-panache.adoc[Hibernate ORM with Panache] is used to store your user identities but you can also use xref:hibernate-orm.adoc[Hibernate ORM] with the `security-jpa` extension.
-Both xref:hibernate-reactive.adoc[Hibernate Reactive] and xref:hibernate-reactive-panache.adoc[Hibernate Reactive with Panache] can be used with the `security-jpa-reactive` extension.
 
-You must also add your preferred database connector library.
-The instructions in this example tutorial use a PostgreSQL database for the identity store.
-====
+.Add the Jakarta Persistence extension to an existing Maven project
 
 * To add the Security Jakarta Persistence extension to an existing Maven project with Hibernate ORM, run the following command from your project base directory:
-
++
+====
 :add-extension-extensions: security-jpa
 include::{includes}/devtools/extension-add.adoc[]
-
+====
 * To add the Security Jakarta Persistence extension to an existing Maven project with Hibernate Reactive, run the following command from your project base directory:
-
++
+====
 :add-extension-extensions: security-jpa-reactive
 include::{includes}/devtools/extension-add.adoc[]
+====
 
 === Verify the quarkus-security-jpa dependency
 
-After you run either of the preceding commands, verify that the `security-jpa` dependency was added to your project build XML file,as follows:
+After you have run either of the preceding commands to create the Maven project, verify that the `security-jpa` dependency was added to your project build XML file.
 
+* To verify the `security-jpa` extension, check for the following configuration:
```

Comment:
Hello @michelle-purcell . This PR is another digital splendor :))
Here is just one little bit that needs to be addressed and you have two options:
1) Use `+` to glue these two snippets together to make them both properly align under the bullet

```
* TextText:
+
[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
.pom.xml
----
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-security-jpa</artifactId>
</dependency>
----
+
[source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
.build.gradle
----
implementation("io.quarkus:quarkus-security-jpa")
----
```

2) Use `+` right after the bullet, then wrap the two snippets in `====` box. This will create an encapsulated unit that is glued to its sticking point :)

```
* TextText:
+
====
[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
.pom.xml
----
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-security-jpa</artifactId>
</dependency>
----

[source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
.build.gradle
----
implementation("io.quarkus:quarkus-security-jpa")
----
====
```

I am horrible when it came to design, so I will keep this one up to you :).